app/Http/Controllers/User/UserController.php

```diff
@@ -278,9 +278,7 @@ public function createNewDataset(Request $request)
     public function userEditDataset(Request $request, int $id)
     {
         $user = auth()->user();
-        $dataset = $user->datasets()->with(['dataitems' => function ($query) {
-            $query->orderBy('id');
-        }])->find($id);
+        $dataset = $user->datasets()->find($id);

```

Comment:
Can you separate this change into a single commit by referencing its related issue number. #188